src/Microsoft.ML.FastTree/GamModelParameters.cs

```diff
@@ -322,6 +335,19 @@ public double[] GetFeatureWeights(int featureIndex)
             return featureWeights;
         }
 
+        /// <summary>
+        /// Get all the binned weights.
```

Comment:
>weights [](start = 31, length = 7)

effects vs weights? Why naming in method vs comment? Try to harmonize.

Reply:
Done.